api/tools/src/main/java/org/pmiops/workbench/tools/SetAuthority.java

```diff
@@ -0,0 +1,79 @@
+package org.pmiops.workbench.tools;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.pmiops.workbench.db.dao.UserDao;
+import org.pmiops.workbench.db.model.User;
+import org.pmiops.workbench.model.Authority;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+/**
+ * See api/project.rb set-authority. Adds or removes authorities (permissions) from users in the db.
+ */
+@SpringBootApplication
+@EnableJpaRepositories("org.pmiops.workbench.db.dao")
+@EntityScan("org.pmiops.workbench.db.model")
+public class SetAuthority {
+
+  private static final Logger log = Logger.getLogger(SetAuthority.class.getName());
+
+  @Bean
+  public CommandLineRunner run(UserDao userDao) {
+    return (args) -> {
+      if (args.length != 3) {
+        throw new IllegalArgumentException(
+            "Expected email_list, authority_list_to_add, authority_list_to_rm. Got " + args);
+      }
+      Set<String> emails = new HashSet();
+      emails.add("mark.fickett@staging.pmi-ops.org");
```

Comment:
This part is changing, right?

Reply:
Yes -- I just hardcoded some values to start trying out the Java before I figured out my command-line parsing strategy. I think what I'd do is something like `Set<String> emails = parseCommaSeparatedString(args[0]);`.